views/js/runner/plugins/tools/areaMasking/areaMasking.js

```diff
@@ -48,6 +52,8 @@ define([
 
             var testRunner = this.getTestRunner();
             var $container = testRunner.getAreaBroker().getContentArea().parent();
+            var testConfig = testRunner.getTestData().config || { plugins : {} };
+            var config     = _.defaults(testConfig.plugins[this.getName()] || {}, defaultConfig);
```

Comment:
Basic plugin config (first argument) will be overwritten here. Maybe it would be better not to change it?
```js
var config = _.defaults({}, testConfig.plugins[this.getName()] || {}, defaultConfig);
```